src/classes/RD_RecurringDonations.cls

```diff
@@ -91,7 +91,8 @@ public class RD_RecurringDonations {
 
     /*******************************************************************************************************
     * @description holds the current custom field mappings used when creating opps for the RD.
-    */ 
+    */
+    @testVisible
```

Comment:
Is `@testVisible` needed if this is already defined a `public`; or should this actually be `private`?

Reply:
Didn't even look at this one as it wasn't part of this refactor.